api/app/Http/Controllers/AuthController.php

```diff
@@ -54,16 +56,17 @@ public function login(Request $request)
             throw new RuntimeException($e->getMessage(), 500, $e);
         }
 
-        return $this->item(new AuthToken(['token' => $token]));
+        return response($transformer->transformItem(new AuthToken(['token' => $token])));
```

Comment:
This point is also to be discussed, as actually I don't like the fact that Laravel makes response transformations under the hood.

I think we need to fire it explicitly each time, so everybody knows where did it come from.